### ISSUE TITLE
Incorrect raising of AmbiguousTime exception

### ISSUE THREAD

**Issue by jozeffrtus** [sdispater/pendulum#162]
Hello,

following snippet incorrectly raises `AmbiguousTime` exception.
```
import pendulum


pendulum.set_transition_rule(pendulum.TRANSITION_ERROR)
print(pendulum.from_timestamp(1529524200).replace(tzinfo='Asia/Dubai'))
```
 
The exception should not be raised, since there is no DST shift in such timezone. Since this issue is related to `pytzdata` timezone binaries compilation I posted it [here](https://github.com/sdispater/pytzdata/issues/3) with broader explanation as well.

Please, can you handle this?

**Comment by sdispater** [comment]
I cannot reproduce this issue anymore.

If you can confirm that this is fixed I will close this issue.

**Comment by jozeffrtus** [comment]
@sdispater the example above is still incorrectly raising the `AmbiguousTime` exception.
```
pip freeze
pendulum==1.4.0
python-dateutil==2.6.1
pytz==2017.3
pytzdata==2017.3.1
six==1.11.0
tzlocal==1.5.1
```

**Comment by sdispater** [comment]
Sorry.

This actually fixed on the `1.x` branch (commit 5a596b0).

The fix will land in the next bugfix release.

**Comment by sdispater** [comment]
This should be fixed in the latest `1.4.1` release :-)

**Comment by jozeffrtus** [comment]
Yes I confirm the issue is fiex in `1.4.1` release. Thank you
